### ISSUE TITLE
please update spring-cloud-gcp-dependencies for 1.0.0.RC1

### ISSUE THREAD

**Issue by n0mer** [spring-cloud/spring-cloud-gcp#821]
Hello, please explain where do you publish dependencies.

latest version of `spring-cloud-gcp-dependencies` is `1.0.0.M1`: https://mvnrepository.com/artifact/org.springframework.cloud/spring-cloud-gcp-dependencies/

https://mvnrepository.com/artifact/org.springframework.cloud/spring-cloud-gcp-starter-pubsub - latest version is 1.0.0.M3

spring-cloud-gcp-starter-config (https://mvnrepository.com/artifact/org.springframework.cloud/spring-cloud-gcp-starter-config) is 1.0.0.M3 as well

**Comment by artembilan** [comment]
That’s wrong: we don’t publish Milestone versions to Maven Central. You need to add to your project a Spring Milestone repository: 
```
<repository>
					<id>spring-milestones</id>
					<name>Spring Milestones</name>
					<url>https://repo.spring.io/libs-milestone-local</url>
					<snapshots>
						<enabled>false</enabled>
					</snapshots>
				</repository>
```

**Comment by n0mer** [comment]
@artembilan https://cloud.spring.io/spring-cloud-gcp/ page states that https://repo.spring.io/libs-milestone should be used for 1.0.0.RC1:

![image](https://user-images.githubusercontent.com/1862997/42137127-450a14a0-7d67-11e8-8cd2-ed0a6017acc4.png)

Does that mean that there is an error @ https://cloud.spring.io/spring-cloud-gcp/ page ?

**Comment by n0mer** [comment]
In other words, is it safe to assume that `1.0.0.RC1` version
* org.springframework.cloud:spring-cloud-gcp-starter-pubsub
* org.springframework.cloud:spring-cloud-gcp-starter-config
* org.springframework.cloud:spring-cloud-gcp-dependencies

can be found at https://repo.spring.io/libs-milestone-local ?

**Comment by artembilan** [comment]
Yeah, sorry, that’s definitely must be https://repo.spring.io/libs-milestone. The `local` one is available as well, but we more use it for our internal purpose.

Right, `config` is not available because it is not GA-ready feature. You need to use `M4` version to poll it for your purpose. We are going to release GA soon and we can’t include `BETA` services to GA.

**Comment by meltsufin** [comment]
Config and Spanner support are currently latest in 1.0.0.M4. They were removed for RC1, which is a bit confusing from the versioning perspective, but all of that should clear up soon when we publish 1.1.0.M1 with Config and Spanner.

**Comment by n0mer** [comment]
@artembilan 

so since 1.0.0 is released

![image](https://user-images.githubusercontent.com/1862997/44627264-12450080-a92b-11e8-8a59-72295c88c789.png)

and there is spring-cloud-gcp-starter-pubsub 1.0.0 

![image](https://user-images.githubusercontent.com/1862997/44627266-24bf3a00-a92b-11e8-8881-e2ab54e4cf15.png)


Are there any plans to update spring-cloud-gcp-starter-config?

![image](https://user-images.githubusercontent.com/1862997/44627269-386aa080-a92b-11e8-91c5-ab0e31e5ce11.png)

**Comment by meltsufin** [comment]
@n0mer Config did not make it into the `1.0.0.RELEASE` because the service itself is not GA yet. However, the latest version of it is available in the Milestones repository: `1.1.0.M1`:

![image](https://user-images.githubusercontent.com/17732492/44664322-e8e6ac00-a9e0-11e8-8b71-9e4ed69a2ccb.png)

Your screenshot is showing the wrong respository. I'm not sure why it's also in the `Spring Plugins` repo.

**Comment by artembilan** [comment]
@meltsufin is right: the Google Cloud Runtime Config is not GA yet: https://cloud.google.com/deployment-manager/runtime-configurator/reference/rest/.

According our policies can't release GA for Spring components when they are based on target BETA service.
